studio/pages/cli/login.tsx

```diff
@@ -0,0 +1,58 @@
+import { NextPage } from 'next'
+import APIAuthorizationLayout from 'components/layouts/APIAuthorizationLayout'
+import { createCliLoginSession } from 'data/cli/login'
+import { useParams } from 'common'
+import { useRouter } from 'next/router'
+import { withAuth } from 'hooks'
+import Connecting from 'components/ui/Loading/Loading'
+import { useEffect, useState } from 'react'
+
+const CliLoginPage: NextPage = () => {
+  const router = useRouter()
+  const { session_id, public_key, token_name, success } = useParams()
+  const [isSuccessfulLogin, setSuccessfulLogin] = useState(false)
```

Comment:
we won't need this then once createCliLoginSession becomes an RQ mutation

Reply:
See my comments above. State is fully controlled by the current url, so mutations here just feels redundant imo

Reply:
ahh okay i see what you mean 👍